README.md

```diff
@@ -54,7 +54,7 @@ Result should be:
 ### Your solution here:
 4.  Fill in the `console.log()`?
 ```js
-console.log()
+console.log("Please" + "squeeze" + "the" + "cheese");
```

Comment:
The required here is to use the variables `firstWord`, `secondWord`, `thirdWord` and `fourthWord` to make this result `"Please squeeze the cheese"`
```suggestion
console.log(firstWord + " " + secondWord + " " + thirdWord + " " + fourthWord);
```